sdks/java/core/src/main/java/com/google/cloud/dataflow/sdk/transforms/windowing/Window.java

```diff
@@ -599,6 +600,33 @@ public void validate(PCollection<T> input) {
       return output.setWindowingStrategyInternal(outputStrategy);
     }
 
+    @Override
+    public void populateDisplayData(DisplayData.Builder builder) {
+      builder
+          .add("windowFn", windowFn.getClass())
+          .include(windowFn);
+
+      if (trigger != null) {
+        builder.add("trigger", trigger.toString());
```

Comment:
Big time. We really need to standardize with Python on the language-independent (shorter) names. In the meantime I will immediately LGTM fixes to such things. Or, even though it is quite trivial, file a JIRA against me as a reminder.